Dockerfile.template

```diff
@@ -45,3 +43,5 @@ COPY --from=expressjs-build-step /build-context .
 COPY expressjs/src src
 
 CMD ["node", "src/index.js"]
+
+ENV NODE_ENV=production
```

Comment:
Is there a particular reason this is down at the bottom? It usually features at the top, like the build step before.